Getting multiple definition error when build ROS-melodic on PI4

When trying to build ros-melodic from source on RPi 4 I am getting error below.
I used build instruction as this
$ sudo ./src/catkin/bin/catkin_make_isolated --install -DCMAKE_BUILD_TYPE=Release --install-space /opt/ros/melodic -j2
```
==> make -j2 in '/home/pi/ros_catkin_ws/build_isolated/qt_gui_cpp'
[  5%] Linking CXX shared library /home/pi/ros_catkin_ws/devel_isolated/qt_gui_cpp/lib/libqt_gui_cpp.so
/usr/bin/ld: CMakeFiles/qt_gui_cpp.dir/ros_pluginlib_plugin_provider.cpp.o:(.rodata+0xea0): multiple definition of `typeinfo name for qt_gui_cpp::RosPluginlibPluginProvider<qt_gui_cpp::PluginProvider>'; CMakeFiles/qt_gui_cpp.dir/recursive_plugin_provider.cpp.o:(.rodata+0x7e4): first defined here
/usr/bin/ld: CMakeFiles/qt_gui_cpp.dir/ros_pluginlib_plugin_provider.cpp.o:(.data.rel.ro+0x20): multiple definition of `typeinfo for qt_gui_cpp::RosPluginlibPluginProvider<qt_gui_cpp::PluginProvider>'; CMakeFiles/qt_gui_cpp.dir/recursive_plugin_provider.cpp.o:(.data.rel.ro+0x0): first defined here
collect2: error: ld returned 1 exit status
make[2]: *** [src/qt_gui_cpp/CMakeFiles/qt_gui_cpp.dir/build.make:292: /home/pi/ros_catkin_ws/devel_isolated/qt_gui_cpp/lib/libqt_gui_cpp.so] Error 1
make[1]: *** [CMakeFiles/Makefile2:307: src/qt_gui_cpp/CMakeFiles/qt_gui_cpp.dir/all] Error 2
make: *** [Makefile:141: all] Error 2
<== Failed to process package 'qt_gui_cpp':
  Command '['/opt/ros/melodic/env.sh', 'make', '-j2']' returned non-zero exit status 2

Reproduce this error by running:
==> cd /home/pi/ros_catkin_ws/build_isolated/qt_gui_cpp && /opt/ros/melodic/env.sh make -j2

Command failed, exiting.

```

What is needed to solve this issue?

--- a/qt_gui_cpp/include/qt_gui_cpp/ros_pluginlib_plugin_provider_for_plugin_providers.h
+++ b/qt_gui_cpp/include/qt_gui_cpp/ros_pluginlib_plugin_provider_for_plugin_providers.h
@@ -35,9 +35,6 @@
 
 #include "plugin_provider.h"
 #include "ros_pluginlib_plugin_provider.h"
-#include "exports.h"
-
-template class QT_GUI_CPP_DECL qt_gui_cpp::RosPluginlibPluginProvider<qt_gui_cpp::PluginProvider>;
 
 namespace qt_gui_cpp
 {

--- a/qt_gui_cpp/include/qt_gui_cpp/ros_pluginlib_plugin_provider_for_plugins.h
+++ b/qt_gui_cpp/include/qt_gui_cpp/ros_pluginlib_plugin_provider_for_plugins.h
@@ -35,9 +35,6 @@
 
 #include "plugin.h"
 #include "ros_pluginlib_plugin_provider.h"
-#include "exports.h"
-
-template class QT_GUI_CPP_DECL qt_gui_cpp::RosPluginlibPluginProvider<qt_gui_cpp::Plugin>;
 
 namespace qt_gui_cpp
 {

